.gitmodules

```diff
@@ -294,3 +294,33 @@
 [submodule "contrib/libdivide"]
 	path = contrib/libdivide
 	url = https://github.com/ridiculousfish/libdivide.git
+[submodule "contrib/aws-crt-cpp"]
+	path = contrib/aws-crt-cpp
+	url = https://github.com/ClickHouse/aws-crt-cpp.git
```

Comment:
Why half of the modules are from `awslabs` and a half from `ClickHouse`?

Reply:
For each module It depends on whether we've changed that module or not.